src/lib/init.ts

```diff
@@ -312,92 +295,228 @@ type InitTSorJSProjectArg = {
   fluenceConfig: FluenceConfig;
 };
 
-const initTSorJSProject = async ({
+async function initTSorJSProject({
   isJS,
   fluenceConfig,
-}: InitTSorJSProjectArg): Promise<void> => {
-  const defaultAquaTSorJSPath = await ensureSrcAquaPath();
-
-  const defaultAquaTSorJSPathRelative = relative(
+}: InitTSorJSProjectArg): Promise<void> {
+  const frontendCompiledAquaPath = await ensureFrontendCompiledAquaPath();
+  const indexFilePath = getFrontendIndexTSorJSPath(isJS);
+  const packageJSONPath = getPackageJSONPath();
+  const frontendSrcPath = getFrontendSrcPath();
+  fluenceConfig.relaysPath = relative(projectRootDir, frontendSrcPath);
+
+  const relativeFrontendCompiledAquaPath = relative(
     projectRootDir,
-    defaultAquaTSorJSPath,
+    frontendCompiledAquaPath,
   );
 
-  const indexFilePath = await ensureSrcIndexTSorJSPath(isJS);
-  const indexFileName = basename(indexFilePath);
+  fluenceConfig[isJS ? "aquaOutputJSPath" : "aquaOutputTSPath"] =
+    relativeFrontendCompiledAquaPath;
+
+  await Promise.all([
+    fluenceConfig.$commit(),
+    writeFile(indexFilePath, getIndexJsContent(isJS), FS_OPTIONS),
+    writeFile(getViteConfigPath(isJS), VITE_CONFIG_CONTENT, FS_OPTIONS),
+    writeFile(packageJSONPath, jsonStringify(getPackageJSON(isJS)), FS_OPTIONS),
+    writeFile(getIndexHTMLPath(), getIndexHTMLContent(isJS), FS_OPTIONS),
+    isJS
+      ? Promise.resolve()
+      : writeFile(getTsConfigPath(), TS_CONFIG_CONTENT, FS_OPTIONS),
+
+    (async () => {
+      return compileToFiles({
+        compileArgs: {
+          filePath: await ensureAquaMainPath(),
+          imports: await ensureAquaImports({
+            maybeFluenceConfig: fluenceConfig,
+          }),
+        },
+        targetType: isJS ? "js" : "ts",
+        outputPath: frontendCompiledAquaPath,
+      });
+    })(),
+  ]);
+}
+
+function getIndexHTMLContent(isJS: boolean) {
+  return `<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="data:,">
```

Comment:
didn't understand this. Maybe we can add fluence icon or something here?

Reply:
we would have to add icon code to template as well, we probably can do it. Chrome always requests favicon.ico even if you remove this and you get 404, this is the simplest way to make chrome shut up

Reply:
inserted base64 encoded image, seems fine